composer.json

```diff
@@ -12,14 +12,15 @@
             "homepage": "https://dunglas.fr"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
should we wait until symfony's released before merging this?

Reply:
Just drop this line, dev versions are already tested in a dedicated CI build.

Reply:
And revert all the `--prefer-stable`

Reply:
But this PR cannot be merged until Symfony 4.3.7 is released.